src/lifecycle/props-init.js

```diff
@@ -15,14 +15,19 @@ function getInitialValue(elem, name, opts) {
   return typeof opts.initial === 'function' ? opts.initial(elem, { name }) : opts.initial;
 }
 
+function getNamespacedPropData(elem, name) {
```

Comment:
Will go with `getPropData()` if you think so to be consistent with the helper functions defined in this context using the `getSomething` nomenclature.